\documentclass[final]{beamer} % beamer 3.10: do NOT use option hyperref={pdfpagelabels=false} !
  %\documentclass[final,hyperref={pdfpagelabels=false}]{beamer} % beamer 3.07: get rid of beamer warnings
  \mode<presentation> { \usetheme{CIPoster} }
  \usepackage[english]{babel}
  \usepackage[latin1]{inputenc}
  \usepackage{amsmath,amsthm, amssymb, latexsym}
  \usefonttheme[onlymath]{serif}
  \boldmath

  \usepackage[orientation=portrait,size=a0,scale=1.1,debug]{beamerposter}                       % e.g. for DIN-A0 poster
    \usepackage{Sweave}
    \usepackage[sort&compress,square,comma,authoryear]{natbib}
\bibliographystyle{plainnat}
  \title[BioTraining]{Bioinformatics Training Opportunities}
  \author[Dunning]{Mark Dunning, Bioinformatics Training Coordinator}
  \institute[C.I]{Bioinformatics Core, Cancer Research UK, Cambridge Institute}
  \date{\today}
  \newcommand{\spaceProp}{0.02}
  \newcommand{\spacer}{\begin{column}{\spaceProp\paperwidth}\end{column}}
  
  \newenvironment{oneCol}{\begin{column}[t]{0.225\paperwidth}}{\end{column}}
  \newenvironment{threeCol}{\begin{column}[t]{0.715\paperwidth}}{\end{column}}
  \setbeamertemplate{caption}[numbered]
\begin{document}
\SweaveOpts{concordance=TRUE}

  \begin{frame}{}
  
            \usebeamercolor{title in headline}{\color{crukpink}\textbf{\large{http://intranet.cri.camres.org/core-facilities/bioinformatics/training-courses}}\\[3ex]}
  
    \begin{columns}[t]
    \spacer{}
    \begin{oneCol}
    \begin{block}{Introduction}
Modern technologies are able to generate vast amounts of data on an unprecedented scale. Therefore numerical and computational competency are increasingly being seen as core skills for bench scientists. 

\end{block}
    \begin{block}{Internal Courses}

  \begin{itemize}
  \item{{\color{crukblue}\textit{Active}}}
  
  These courses are designed to help you make the most of your interactions with the Genomics and Bioinformatics Cores. They will help you to formulate your experimental design question, and give you a basic grounding in statistics.
    \begin{itemize}
      \item{{\color{crukpink}{An Introduction to Statistical Analysis using R Commander}}}
      \item{{\color{crukblue}{Introduction to Experimental Design}}}
    \end{itemize}
  
  \item{\color{crukblue}\textit{Under development}}
    \begin{itemize}
    \item{{\color{crukpink}{Visualisation and interpretation of Bioinformatcs data using Genome Browsers}}}
    \item{{\color{crukblue}{Graphics and data manipulation in R}}}
    \item{{\color{crukpink}{Further statistics}}}
    \item{{\color{crukblue}{Introduction to Unix}}}
    \item{{\color{crukpink}{ChIP-seq Data Analysis}}}
    \item{{\color{crukblue}{RNA-Seq Data Analysis}}}
    \end{itemize}
  \end{itemize}

    \end{block}
    
    
    \begin{block}{External Courses}
    All members of The University of Cambridge are eligible for courses hosted in the Department of Genetics. A list of current courses can be obtained from {\color{crukpink}tinyurl.com/biocourses}. Several courses that involve C.I. staff as demonstrators, or might be of particular interested, are listed below
  \begin{itemize}
  \item{\color{crukblue}{Beginner}}
  \begin{itemize}
    \item{{\color{crukpink}{An Introduction to Solving Biological Problems with Python}}}
    \item{{\color{crukblue}{An Introduction to Solving Biological Problems with PERL}}}
    \item{{\color{crukpink}{An Introduction to Solving Biological Problems with R}}}
    \item{{\color{crukblue}{Analysing mapped NGS data with SeqMonk}}}
    \item{{\color{crukpink}{Galaxy: Data Manipulation and Visualisation}}}
    \item{{\color{crukblue}{Whole Exome Sequencing and RNA-sequence data analysis}}}
    \item{{\color{crukpink}{Introductory Statistics and Experimental Design for Genomics}}}
  \end{itemize}
  \item{\color{crukblue}{Intermediate}}
    \begin{itemize}
    \item{{\color{crukpink}{Analysis of High-throughput sequencing data with Bioconductor}}}
    \item{{\color{crukblue}{Microarray Analysis with Bioconductor}}}
    \end{itemize}
  \item{\color{crukblue}{Advanced}}
    \begin{itemize}
    \item{{\color{crukpink}{A Practical Introduction to Good Programming Practices}}}
    \item{{\color{crukblue}{R object-oriented programming and package development}}}
    \end{itemize}
  
  \end{itemize}
    
    \end{block}

    
    \begin{block}{Course Materials}
    Materials from our internal courses will be hosted on the C.I. sharepoint: {\color{crukpink}{sharepoint.cri.camres.org/sites/bioinformatics/Public/}}
    \end{block}
    
    \begin{block}{Interested in becoming a trainer?}
    We are always keen to recruit Bioinformaticians to the training roster. If you think you can contribute to one of our existing courses, or can spot a gap in our schedule, please let us know!
    \end{block}
    
    \begin{block}{Acknowledgements}
    The current roster of course developers and presenters is as follows;
    {\color{crukblue}{Ines de Santiago}}, {\color{crukpink}{Henry Farmery}},{\color{crukblue}{Sarah Leigh Brown}}, {\color{crukpink}{Suraj Menon}}, {\color{crukblue}{Anne Pajon}}, {\color{crukpink}{Oscar Rueda}}, {\color{crukblue}{Roslin Russell}}, {\color{crukpink}{Shamith Samarajiwa}}, {\color{crukblue}{Mike Smith}},{\color{crukpink}{Rory Stark}} {\color{crukblue}{Jing Su}}, {\color{crukpink}{Sarah Vowler}}
    \end{block}
        \begin{block}{References}
      \bibliography{poster}
  		\normalsize

    \end{block}
    \end{oneCol}

    \spacer{}
    \begin{threeCol}
    
    \begin{block}{Aims}
    \begin{itemize}
    \item{Ensure that researchers leave the institute with competency in numerical and computational analysis. We focus on core skills in {\color{crukpink}{statistics}}, {\color{crukblue}{experimental design}}, {\color{crukpink}{R}}, {\color{crukblue}{Reproducible Research}}  and {\color{crukpink}{unix}}.}
    \item{Increase researcher independence by promoting interfaces such as IGV, Galaxy and UCSC for routine Bioinformatics analysis and data exploration}
    \item{Monitor and advertise Bioinformatics training opportunities that arise in the University}
    \end{itemize}
    \end{block}
    \begin{block}{Experimental Design and Statistics}



  
        {\color{crukpink}{\textit{"To consult the statistician after an experiment is finished is often merely to ask him to conduct a post mortem examination. He can perhaps say what the experiment died of." - R.A. Fisher}}}
        
This quote by Fisher sums up why experimental design is a crucial part of the experimental process. For this reason, our training course on Experimental Design is \textbf{compulsory} for all new PhD students. The course is also run regularly for other researchers at C.I.\\


Statistics teaching is done using the R Commander GUI; a user-friendly way of doing statistical analysis in R (\cite{Rcmdr}). Therefore the focus of this training is on the statistical concepts, rather than wrestling with the R language. In this course we introduce the basics of statistics and hypothesis testing. 
Material from our experimental design and statistics courses are available on {\color{crukblue}{Sharepoint}}. Individual consultations with the Statistics team can be arranged by emailing; {\color{crukpink}{{\tt cristatsclinic@cruk.cam.ac.uk}}}\vspace{1cm}


 
    \end{block}
    \begin{block}{R Training}
    \begin{columns}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/NYTimes_R_Article.png}
    \caption{{\color{crukblue}{R is gaining press attention {\color{crukpink}{(\cite{i1697})}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/cyclejourneys.png}
    \caption{{\color{crukblue}{R can produce high-quality graphics {\color{crukpink}{(\cite{Cheshire:2012})}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
        \includegraphics{images/osqa.png}
      \caption{{\color{crukblue}{R forum for C.I. staff - {\color{crukpink}{http://bioinf-qa001/}}}}}
        \end{figure}

        \end{oneCol}
    \end{columns}
The R programming language is our language of choice for data analysis. Not only is it open-source and freely-available, but it is capable of generating publication-quality images and has a large user-base amongst the scientific community. It also promotes the practice of \textbf{Reproducible Research}.
However, it comes with a steep learning-curve especially for those without prior exposure to command-line tools. Although it requires an investment of time, it is ultimately rewarding and a valuable skill to acquire. We are taking the following steps to help with the transition from GraphPad Prism.\vspace{1cm}

\begin{itemize}
\item{Promote availability of R classes at The Department of Genetics}
\item{Provide extra material to support the course in Genetics}
\item{Bespoke training on R graphics with online materials}
\item{Worked-examples of how to conduct typical GraphPad Prism analyses in R}
\item{Support forum for C.I. staff - {\color{crukpink}\textbf{\large{\tt http://bioinf-qa001/}}}}
\end{itemize}

    \end{block}
    
\begin{block}{Data Visualisation and Exploration}

   \begin{columns}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/GalaxyScreenshot.png}
    \caption{{\color{crukblue}{Galaxy {\color{crukpink}{{\tt https://usegalaxy.org/}}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/igvfigure1.jpg}
    \caption{{\color{crukblue}{Integrative Genomics Viewer (IGV) {\color{crukpink}{\tt http://www.broadinstitute.org/igv/}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
        \includegraphics{images/ucsc007.png}
        \caption{{\color{crukblue}{U.C.S.C Genome Browser {\color{crukpink}{\tt https://genome.ucsc.edu/}}}}}
        \end{figure}
        \end{oneCol}
                
    \end{columns}
Arguably the best way to develop an understanding of high-throughput data is to get into the habit of visualising the data. There are many tools that provide the opportunity to do this. We are currently developing introductory and advanced courses to introduce such tools. Researchers will be show how to load their own data into these tools and overlay with other genomic 'tracks' and perform simple operations. \vspace{1cm}

\begin{itemize}
\item{The Galaxy browser}
\item{The Integrative Genomics Viewer (IGV)}
\item{The UCSC genome browser}
\end{itemize}

\end{block}

\begin{block}{Analysis of High-throughput data}
   \begin{columns}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/volcanoplot.png}
    \caption{{\color{crukblue}{Volcano Plot showing differentially expressed genes}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/Heatmap.png}
    \caption{{\color{crukblue}{Heatmap representing relationships between biological samples}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
        \includegraphics{images/cir.png}
        \caption{{\color{crukblue}{Visualising sequencing data in Bioconductor}}}
        \end{figure}
        \end{oneCol}
                
    \end{columns}
Along with other Bioinformaticians at C.I., the Bioinformatics Core has made a significant contribution to training on the analysis of Microarray and Next-Generation Sequencing Data. These courses are hosted at the Genetics Department and aimed at those with some experience of R and keen on doing their own analysis. Not only do we provide practical experience of analysing data, but also offer insight into the methodology. Software from the Bioconductor project is used in the courses ({\color{crukpink}{www.bioconductor.org}}). We will also offer courses tailored around the output of our RNA-seq and ChIP-seq pipelines. These will focus on how to interpret the results delivered by the Bioinformatics Core.
\end{block}

    
    
    \end{threeCol}
    \spacer{}
    \end{columns}
  \end{frame}
  \end{document}
