\documentclass[final]{beamer} % beamer 3.10: do NOT use option hyperref={pdfpagelabels=false} !
  %\documentclass[final,hyperref={pdfpagelabels=false}]{beamer} % beamer 3.07: get rid of beamer warnings
  \mode<presentation> { \usetheme{CIPoster} }
  \usepackage[english]{babel}
  \usepackage[latin1]{inputenc}
  \usepackage{amsmath,amsthm, amssymb, latexsym}
  \usefonttheme[onlymath]{serif}
  \boldmath

  \usepackage[orientation=portrait,size=a0,scale=1.1,debug]{beamerposter}                       % e.g. for DIN-A0 poster
    \usepackage{Sweave}
    \usepackage[sort&compress,square,comma,authoryear]{natbib}
\bibliographystyle{plainnat}
  \title[BioTraining]{Bioinformatics Training Opportunities}
  \author[Dunning]{Mark Dunning, Bioinformatics Training Coordinator}
  \institute[C.I]{Bioinformatics Core, Cancer Research UK, Cambridge Institute}
  \date{\today}
  \newcommand{\spaceProp}{0.02}
  \newcommand{\spacer}{\begin{column}{\spaceProp\paperwidth}\end{column}}
  
  \newenvironment{oneCol}{\begin{column}[t]{0.225\paperwidth}}{\end{column}}
  \newenvironment{threeCol}{\begin{column}[t]{0.715\paperwidth}}{\end{column}}
  \setbeamertemplate{caption}[numbered]
\begin{document}
\SweaveOpts{concordance=TRUE}

  \begin{frame}{}
  
            \usebeamercolor{title in headline}{\color{crukpink}\textbf{\large{http://bioinformatics-core-shared-training.github.io/}}\\[3ex]}
  
    \begin{columns}[t]
    \spacer{}
    \begin{oneCol}
    \begin{block}{Introduction}
Modern technologies are able to generate vast amounts of data on an unprecedented scale. Therefore numerical and computational competency are increasingly being seen as core skills for bench scientists. 

\end{block}

    
    
    \begin{block}{University-run Courses}
    All members of The University of Cambridge are eligible for courses hosted in the Department of Genetics. A list of currently-scheduled courses can be obtained from {\color{crukpink}http://tiny.cc/biocourses}. Several courses that involve C.I. staff as demonstrators, or might be of particular interested, are listed below
  \begin{itemize}
  \item{\color{crukblue}{Beginner}}
  \begin{itemize}
    \item{{\color{crukpink}{An Introduction to Solving Biological Problems with Python}}}
    \item{{\color{crukblue}{An Introduction to Solving Biological Problems with PERL}}}
    \item{{\color{crukpink}{An Introduction to Solving Biological Problems with R}}}
    \item{{\color{crukpink}{An Introduction to MATLAB for biologists}}}
    \item{{\color{crukblue}{Analysing mapped NGS data with SeqMonk}}}
    \item{{\color{crukpink}{Galaxy: Data Manipulation and Visualisation}}}
    \item{{\color{crukblue}{Galaxy: RNA-seq and ChIP-seq data analysis }}}
    \item{{\color{crukpink}{Whole Exome Sequencing and RNA-sequence data analysis}}}
    \item{{\color{crukblue}{Introduction to genome variation analysis using NGS}}}
  \item{{\color{crukpink}{Using the Ensembl Genome Browser}}} 
  \end{itemize}
  \item{\color{crukblue}{Intermediate}}
    \begin{itemize}
    \item{{\color{crukpink}{Analysis of High-throughput sequencing data with Bioconductor}}}
    \item{{\color{crukblue}{Microarray Analysis with Bioconductor}}}
    \end{itemize}
  \item{\color{crukblue}{Advanced}}
    \begin{itemize}
    \item{{\color{crukpink}{A Practical Introduction to Good Programming Practices}}}
    \item{{\color{crukblue}{R object-oriented programming and package development}}}
    \end{itemize}
  
  \end{itemize}
    
    \end{block}

    

    
    \begin{block}{Interested in becoming a trainer?}
    We are always keen to recruit Bioinformaticians to the training roster. If you think you can contribute to one of our existing courses, or can spot a gap in our schedule, please let us know!
    \end{block}
    
    \begin{block}{Acknowledgements}
    The current roster of course developers and presenters is as follows;
    \begin{itemize}
    \item{{\color{crukblue}{Chandu Chilamakuri}}}
    \item{{\color{crukpink}{Ines de Santiago}}}
    \item{{\color{crukblue}{Henry Farmery}}}
    \item{{\color{crukpink}{Peter MacCallum}}}
    \item{{\color{crukblue}{Suraj Menon}}}
    \item{{\color{crukpink}{Marc O'Brien}}}
    \item{{\color{crukblue}{Anne Pajon}}}
    \item{{\color{crukpink}{Bernard Pereira}}}
    \item{{\color{crukblue}{Oscar Rueda}}}
    \item{{\color{crukpink}{Rory Stark}}}
    \item{{\color{crukblue}{Jing Su}}}
    \item{{\color{crukpink}{Sarah Vowler}}}
    \end{itemize}
    \end{block}
        \begin{block}{References}
      \bibliography{poster}
  		\normalsize

    \end{block}

    
    \end{oneCol}

    \spacer{}
    \begin{threeCol}
    
    \begin{block}{Aims}
    \begin{itemize}
    \item{Ensure that researchers leave the institute with competency in numerical and computational analysis. We focus on core skills in {\color{crukpink}{statistics}}, {\color{crukblue}{experimental design}}, {\color{crukpink}{R}}, {\color{crukblue}{data visualisation}} and {\color{crukpink}{unix}}.} These are discussed briefly in more detail below.
    \item{Increase researcher independence by promoting interfaces such as IGV, Galaxy and UCSC for routine Bioinformatics analysis and data exploration}
    \item{Monitor and advertise Bioinformatics training opportunities that arise in the University}
    \end{itemize}
    \end{block}
    
    \begin{block}{Bioinformatics Courses at CI - 2015 Schedule}
    {\color{crukpink}{Seminar room 215}} in Cancer Research Uk Cambridge Institute is block-booked for training two days a month. The courses in this room are run in collaboration with the Bioinformatics Training Programme at the Department of Genetics. However, priority to attend the course and first notification of courses will be given to CCC members. Participants are expected to bring their own laptops and instructions are provided on how to install the relevant software.\\
    
    
    \begin{itemize}
    \item{\color{crukblue}{Monday 19th October - Further Staistical Analysis using R}}
    \item{\color{crukpink}{Monday 16th November - Experimental Design}}
    \item{\color{crukblue}{Tuesday 17th November - Introduction to Statistical Analysis}}
    \item{\color{crukpink}{Monday 30th November - Basic Unix}}
    \item{\color{crukblue}{Friday 4th December - Scientific Figure Design (live webcast from Department of Genetics)}}
    \end{itemize}
    \end{block}
    
    \begin{block}{Unix}
    The Unix shell has existed since the early days of computers, and yet is still the preferred way to run many popular Bioinformatics tools. The concept of typing commands to run programs is often alien to those that are used-to a GUI interface, so we aim to take the novice and turn them into a beginning Linux user. We describe the Linux environment so they can start to utilize command-line tools and feel comfortable using a text-based way of interacting with a computer. Teaching is done via a {\color{crukpink}{'virtual machine'}} so that participants can experiment without making permanent changes to their own machine.
    \end{block}
    
    \begin{block}{R}
    \begin{columns}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/NYTimes_R_Article.png}
    \caption{{\color{crukblue}{R is gaining press attention {\color{crukpink}{(\cite{i1697})}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/cyclejourneys.png}
    \caption{{\color{crukblue}{R can produce high-quality graphics {\color{crukpink}{(\cite{Cheshire:2012})}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
        \includegraphics{images/rstudio-windows.png}
      \caption{{\color{crukblue}{Rtsudio promotes reproducible research}}}
        \end{figure}

        \end{oneCol}
    \end{columns}
The R programming language is our language of choice for data analysis. Not only is it open-source and freely-available, but it is capable of generating publication-quality images and has a large user-base amongst the scientific community. 
However, it comes with a steep learning-curve especially for those without prior exposure to command-line tools. Although it requires an investment of time, it is ultimately rewarding and a valuable skill to acquire. \\

R, and especially the RStudio interface, also promotes the practice of {\color{crukpink}{Reproducible Research}}. We are actively-engaged in restructuring the University's long-standing R course so that a key component is the generation of reproducible analyses and reports.
\end{block}

\begin{block}{Experimental Design \& Statistics}
        {\color{crukpink}{\textit{"To consult the statistician after an experiment is finished is often merely to ask him to conduct a post mortem examination. He can perhaps say what the experiment died of." - R.A. Fisher}}}\\
        
This popular quote by Fisher sums up why experimental design is a crucial part of the experimental process. For this reason, our training course on Experimental Design is {\color{crukpink}{highly-recommended for all new PhD students}}.\\

For statistics training, we have developed a series of online tools using {\color{crukpink}{Shiny (http://shiny.rstudio.com/)}}; a framework for developing web applications using R. Therefore the focus of this training is on the statistical concepts, rather than wrestling with the R language. In this course we introduce the basics of statistics and hypothesis testing so that participants can conduct their own simple statistical testing, and be able to interpret published results. 


    \end{block}
    
\begin{block}{Data Visualisation and Exploration}

   \begin{columns}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/GalaxyScreenshot.png}
    \caption{{\color{crukblue}{Galaxy {\color{crukpink}{{\tt https://usegalaxy.org/}}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
    \includegraphics{images/igvfigure1.jpg}
    \caption{{\color{crukblue}{Integrative Genomics Viewer (IGV) {\color{crukpink}{\tt http://www.broadinstitute.org/igv/}}}}}
    \end{figure}
    \end{oneCol}
    %\spacer{}
    \begin{oneCol}
    \begin{figure}
        \includegraphics{images/ucsc007.png}
        \caption{{\color{crukblue}{U.C.S.C Genome Browser {\color{crukpink}{\tt https://genome.ucsc.edu/}}}}}
        \end{figure}
        \end{oneCol}
                
    \end{columns}
Arguably the best way to develop an understanding of high-throughput data is to get into the habit of visualising the data. There are many tools that provide the opportunity to do this. We are currently developing introductory and advanced courses to introduce such tools. Researchers will be show how to load their own data into these tools and overlay with other genomic 'tracks' and perform simple operations in the most-popular tools. \vspace{1cm}


\end{block}

        \begin{block}{Course Materials}
    Our courses are based-on open-source and freely-available software. Materials from our courses are developed and shared via Github: {\color{crukpink}{https://github.com/bioinformatics-core-shared-training}}. We also have collaborations and share material with the Bioinformatics Training Programme (Department of Genetics) and MRC Clinical Sciences Centre (London). 
       \begin{columns}
           \begin{oneCol}
    \begin{figure}
        \end{figure}
        \end{oneCol}
        \begin{oneCol}
    \begin{figure}
        \includegraphics[height=4cm,width=9.7cm]{images/GitHub_Logo.png}
        \end{figure}
        \end{oneCol}

    \begin{oneCol}
    \begin{figure}
    \includegraphics[height=3.50cm,width=4.2cm]{images/Octocat.jpg}
        \end{figure}
        \end{oneCol}
    \end{columns}
    \end{block}

    
    
    \end{threeCol}
    \spacer{}
    \end{columns}
  \end{frame}
  \end{document}
